### PR TITLE
Fix spelling mistakes and typos in blog post

### PR DESCRIPTION
**Pull request by juliaogris**

### CODE REVIEW

File: docs/blog/setup.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
-The Sysl open source setup
-======================
+The Sysl open source set-up
```
Comment (anzdaddy) [inline docs/blog/setup.md]: http://www.future-perfect.co.uk/grammar-tip/is-it-setup-set-up-or-set-up/

Comment (juliaogris) [reply docs/blog/setup.md]: sublimes british en dict does not know the word setup. sorry!

Comment (juliaogris) [reply docs/blog/setup.md]: done.

**Comment by juliaogris** [comment]
@anzdaddy, could you please re-approve

**Pull request merged**